components/edit/configure/customTool.tsx

```diff
@@ -378,14 +378,18 @@ const CustomTool: React.FC<ConfigureProps> = ({ tool }) => {
                   )}
                 </div>
               </div>
-              <div className="h-full overflow-y-auto">
+              <div className="h-full overflow-y-auto pt-14">
                 <ChatContextProvider
                   initialScript={scriptPath}
                   enableThread={false}
                   initialSubTool={customTool.name}
                   initialScriptId={`${scriptId}`}
                 >
-                  <Chat messagesHeight="h-[93%]" />
+                  <Chat
+                    disableInput={!customTool.chat}
+                    noChat={!customTool.chat}
```

Comment:
This indicates that `disableInput` and `noChat` are the same. Do we need both?

Reply:
They're not quite the same, no. `noChat` is special. However, I can make it so we don't need to pass `disableInput` here too.

Reply:
Right here, they are exactly the same. Sounds like we don't need both. I will formally request that one of them gets removed here.

Reply:
I'm going to +1 to what Tyler is saying here. It seems that these 2 props affect the `Chat` component in a non-related way. Seeing that `Chat` is used in 4 different places leads me to believe that different variations of these props are necessary and actually make the component more reusable. 

@tylerslaton let me know if I'm off here

Reply:
Yeah something like that. `noChat` essentially just means the script has chat disabled. Therefor no chat bar should appear but we want a "restart" button.